Sources/LocalReceiptParsing/BasicTypes/AppleReceipt.swift

```diff
@@ -90,6 +90,15 @@ extension AppleReceipt {
         )
     }
 
+    /// Returns the most recent subscription (see `InAppPurchase.isActiveSubscription`).
+    var lastPurchasedSubscription: InAppPurchase? {
```

Comment:
❤️

Reply:
this doesn't specify that it's _active_, though, how about `mostRecentActiveSubscription` or `lastPurchasedActiveSubscription`?